key-share/CHANGELOG.md

```diff
@@ -16,5 +27,7 @@
 [#82]: https://github.com/dfns/cggmp21/pull/82
 
 ## v0.1.0
+**YANKED**: this release is yanked because it had an issue with key share (de)serialization
+that was addressed in v0.2.2
```

Comment:
It was never released in the first place :D

Reply:
v0.1? it was https://crates.io/crates/key-share/0.1.0

Reply:
actually, v0.2.1 was never released...